src/test/streaming_test.cpp

```diff
@@ -1301,6 +1301,35 @@ node {
     ASSERT_EQ(executor.inferStream(this->firstRequest, this->stream), StatusCode::OK);
 }
 
+TEST_F(StreamingTest, FirstRequestRestrictedParamName) {
```

Comment:
what happens if the second request comes with restricted param name?

Reply:
Second request parameters are not deserialized. Only the first request can deliver those.